src/Common/src/Common.Http/HttpClientHelper.cs

```diff
@@ -67,6 +67,14 @@ public static HttpClient GetHttpClient(bool validateCertificates, HttpClientHand
             return client;
         }
 
+        public static HttpClient GetHttpClient(HttpMessageHandler handler, int timeout = 15)
```

Comment:
In lieu of XML docs ...

```
, int timeoutMillis = 15)
```

Reply:
oh, wow... this should probably not be 15 ms, that seems a bit short for an http timeout!